cadence/contracts/bridge/CrossVMNFT.cdc

```diff
@@ -50,24 +59,38 @@ access(all) contract CrossVMNFT {
     /// See discussion https://github.com/onflow/flow-nft/pull/126#discussion_r1462612559 where @austinkline raised
     /// differentiating IDs in a minimal interface incorporated into the one below
     ///
-    access(all) resource interface EVMNFT : NonFungibleToken.NFT {
-        access(all) let evmID: UInt256
-        access(all) let name: String
-        access(all) let symbol: String
-        access(all) fun tokenURI(): String
-        access(all) fun getEVMContractAddress(): EVM.EVMAddress
+    access(all)
+    resource interface EVMNFT : NonFungibleToken.NFT {
+        access(all)
```

Comment:
I agree, I prefer the single line formatting. Figured I'd try this out for consistency with the EVM contract formatting but with these shorter lines it looks a bit wonky

Reply:
I like to have `access(all)` on a separate line when the function signature is too long. Only works when there is a newline between each "section" (declaration, function, etc). 
Anyway I don't think you should do it on every occurrence (esp var declarations) because its a rule.